spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,40 @@
+require "rails_helper"
+
+RSpec.describe SessionsController, type: :controller do
+  before do
+    DatabaseCleaner.strategy = :transaction
+    OmniAuth.config.test_mode = true
+  end
+  describe "#create" do
+    before do
+      set_valid_omniauth
+      request.env["omniauth.auth"] = OmniAuth.config.mock_auth[:google_oauth2]
+    end
+    it "should successfully create a user" do
+      expect { post :create,
+        provider: :google_oauth2 }.to change{ User.count }.by(1)
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Expression at 15, 34 should be on its own line.<br>Space missing to the left of {.